scripts/seed.js

```diff
@@ -1267,13 +1360,56 @@ async function createMembers() {
         active: false
     });
 
+    const netcomMember = await User.create({
```

Comment:
These new users also should be added to a local so we can actually use them. All users should also be agreeing to the privacy policy in the seeder, but that can be done in a future PR